content/guide/core/tracing.md

```diff
@@ -0,0 +1,250 @@
+---
+title: Tracing
+---
+
+The Trace class allows you to:
+
+- control the [categories of messages](#addcategories-and-setcategories) you log to the console
+- [disable](#disable) multiple messages logging at once
+- [handle errors](/guide/error-handling)
+
+## Using Trace
+
+The following 3 steps outline the basic usage of the Trace class.
+
+The first two step should be executed earlier in your applicaton, usually in the [app.ts](/project-structure/src/main-js-ts) file.
+
+1. Specify atleast one category to trace:
+
+```ts
+Trace.setCategories("category"))
+// or
+Trace.setCategories(Trace.categories.concat("category1","category2"));
+//or
+Trace.addCategories("categ1, categ2")
+```
+If you don't set any category or the category you pass to `Trace.write()` is not one of the registered, `Trace.write`'s message won't be logged.
+
+2. Enable tracing
+
+```ts
+Trace.enable()
+```
+
+3. Call the `Trace.write()` where you would call `console.log()` passing it a message,a category name and optionally, a message type. The category should be one of those you set with `Trace.setCategories()`
```

Comment:
```suggestion
3. Instead of using console.log(), use Trace.write() to log your messages. Pass the message content, a category name, and optionally, a message type as arguments to Trace.write(). Make sure that the category you provide is one of the categories you have previously set using Trace.setCategories().
```